<launch>
	<arg name="dev" default="/dev/input/js0"/>
	<arg name="is_publish_path" default="true"/>
	<arg name="is_log_path" default="false"/>
	<arg name="logs_directory" default="$(find jetson_car_teleop)/logs"/>
	<arg name="path_file"/>

    <param name="dev" value="$(arg dev)"/>
    <param name="is_publish_path" value="$(arg is_publish_path)"/>
    <param name="logs_directory" value="$(arg logs_directory)" if="$(arg is_log_path)"/>

	<node name="joystick" pkg="joy" type="joy_node"/>
	<node name="path_publisher" pkg="jetson_car_teleop" type="path_pub.py" args="$(arg path_file)"/>
	<node name="rviz" pkg="rviz" type="rviz" args="-d $(find jetson_car_teleop)/rviz/path.rviz" />
	<node name="odometry_logger" pkg="jetson_car_teleop" type="odometry_logger.py" output="screen"/>
</launch>
